Description: add a missing lib to the list given to the linker
Author: Sylvestre Ledru
Forwarded: http://bugzilla.scilab.org/show_bug.cgi?id=15452

--- a/scilab/Makefile.am
+++ b/scilab/Makefile.am
@@ -110,6 +110,7 @@
 if NEED_JAVA
 scilab_bin_LDADD += \
 			$(top_builddir)/modules/console/libsciconsole.la \
+			$(top_builddir)/modules/console/libsciconsole-minimal.la \
 			$(top_builddir)/modules/jvm/libscijvm.la \
 			$(top_builddir)/modules/commons/libscicommons.la
 else
